### ISSUE TITLE
Potential Inaccuracy in MSELoss Implementation

### ISSUE THREAD

**Issue by ykx3** [Tokisakix/Terox#1]
I noticed that the implementation of the MSELoss function in `project/tensor/function.py` might not be fully aligned with the standard definition of Mean Squared Error (MSE). Specifically, the current implementation does not average the squared differences.
```python
def MSELoss(inputs:Tensor, labels:Tensor) -> Tensor:
    loss = (inputs - labels) * (inputs - labels)
    return loss
```

**Comment by Tokisakix** [comment]
Thank you for raising the issue!

Indeed, the implementation of MSELoss here is somewhat different from the original version, and the implementation of pure matrix is adopted here, because terox does not support the broadcast mechanism of tensor at present, and the gradient calculation of reduce operations such as sum and mean is not perfect at present.
